fs/io_zenfs.h

```diff
@@ -65,6 +66,7 @@ class ZoneFile {
   IOStatus SetWriteLifeTimeHint(Env::WriteLifeTimeHint lifetime);
   std::string GetFilename();
   void Rename(std::string name);
+  time_t GetFileCTime();
```

Comment:
Call this FileCreationTime, no abbreviations please

Reply:
OK